package.json

```diff
@@ -42,8 +42,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test .",
     "test:coverage": "react-scripts test . --coverage --watchAll=false",
-    "lint": "eslint src/**/*.js",
-    "lint:fix": "eslint src/**/*.js --fix",
+    "lint": "eslint src/**/*.js src/*.js",
```

Comment:
I had to explicitly add the glob pattern for .js files directly in src. Locally it ran fine without this, but for some reason the github action needs it explicitly stated.